iosTests/prepareios.js

```diff
@@ -22,7 +22,8 @@ console.log('=== Installing sdk dependencies');
 execSync('node ./updatesdk.js', {stdio: [0,1,2]});
 
 console.log('=== Installing pod dependencies');
-execSync('rm .xcode.env.local', {stdio:[0,1,2], cwd:'ios'});
+const nodePath = execSync('command -v node', { encoding: 'utf-8' }).trim();
+execSync(`echo export NODE_BINARY=${nodePath} > .xcode.env`, {stdio:[0,1,2], cwd:'ios'});
```

Comment:
Otherwise getting a build error:

> You need to configure your node path in the `".xcode.env" file` environment.  You can set it up quickly by running:  `echo export NODE_BINARY=$(command -v node) > .xcode.env`  in the ios folder. This is needed by React Native to work correctly.  We fallback to the DEPRECATED behavior of finding `node`. This will be REMOVED in a future version.  You can read more about this here: https://reactnative.dev/docs/environment-setup#optional-configuring-your-environment